examples/platforms/simulation/ble.c

```diff
@@ -87,6 +88,16 @@ static void deinitFds(void)
     }
 }
 
+otError otPlatBleGetAdvertisementBuffer(otInstance *aInstance, uint8_t **aAdvertisementBuffer)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+    static uint8_t sAdvertisemntBuffer[OT_TCAT_ADVERTISEMENT_MAX_LEN];
+
+    *aAdvertisementBuffer = sAdvertisemntBuffer;
```

Comment:
```suggestion
    *aAdvertisementBuffer = sAdvertisementBuffer;
```

Sorry for the late comment!  This looked like possibly a typo - or was it intentional? If `aAdvertisementBuffer` uses the extra "e" in the name, then one would expect also `sAdvertisementBuffer` as a name.    Or else, both in the short form.

Reply:
fixed.